flink-table/flink-table-common/src/main/java/org/apache/flink/table/types/logical/utils/LogicalTypeMerging.java

```diff
@@ -302,15 +302,19 @@ public static LogicalType findSumAggType(LogicalType argType) {
      *
      * <p>https://docs.microsoft.com/en-us/sql/t-sql/data-types/precision-scale-and-length-transact-sql
      *
-     * <p>The rules although inspired by SQL Server they are not followed 100%, instead the approach
-     * of Spark/Hive is followed for adjusting the precision.
+     * <p>The rules (although inspired by SQL Server) they are not followed 100%, instead the
```

Comment:
the `they` should have been dropped as well ;)

Reply:
no worries, fixed.